### PR TITLE
rename-migration-files

### PR DESCRIPTION
**Pull request by Humungus-Fungus**
For issue #53.
The naming pattern has changed to using the date the file was created.

### CODE REVIEW

File: backend/alembic.ini [hunk 1]
```diff
@@ -41,6 +41,8 @@ prepend_sys_path = .
 
 sqlalchemy.url = %(sqlalchemy_url)s
 
+file_template = %%(year)d, %%(month).2d, %%(day).2d
```
Comment (galbwe) [inline backend/alembic.ini:44]: Hey, thanks for contributing! Here are a few suggestions:

1. Please remove the spaces in the template. These will be generated filenames and should not contain spaces. The default filenames use underscores (`_`) as a separator, so we might as well continue using those.
2. Can you include the slug after the date? That will help distinguish between migrations made on the same day.
3. Please remove the commented out `file_template=` line (~line 8 or so)
4. Once that's done, can you rename the existing migration scripts to follow the new pattern? They should be in the `versions` directory. You'll also need to change the references to other revisions. These are the lines that look like:
```python
# revision identifiers, used by Alembic.
revision = "063a9e5bc287"
down_revision = "37ad66f1e7a5"
branch_labels = None
depends_on = None
```

Comment (Humungus-Fungus) [reply backend/alembic.ini]: Okay, I implemented your suggestions.

**Comment by galbwe** [comment]
@Humungus-Fungus looks like you need to resolve conflicts in pinned node dependencies. To do that, delete and reinstall the packages with

```
cd frontend/cfd-partner-finder-frontend
rm package-lock.json
npm i
```

**Comment by Humungus-Fungus** [comment]
Okay I managed to fix the conflict with the package-lock.json file, but I'm not sure how to fix this one

**Comment by galbwe** [comment]
I can't see what the conflicts are, but you probably need to resolve them by keeping the files with the new names. I was able to merge your branch into the main branch locally without any problems, so you might need to pull the latest changes to main and merge them into your branch.

**Comment by Humungus-Fungus** [comment]
Alright, everything looks good now! :)

**Comment by galbwe** [comment]
@Humungus-Fungus thanks! Merged it!

**Pull request merged**